README.md

```diff
@@ -1 +1,359 @@
-# starter-kit-se-deployment-notes
\ No newline at end of file
+# Deployment of GDI starter kit
+
+This repo is a collection of scripts and documentation, produced during the deployment of the GDI starter kit in the Swedish node. However, it shouldn't be used as a standalone guide and it probably lacks some steps, which were not documented during the deployment.
+
+The deployment described here was done in one VM (virtual machine) and the services were deployed in docker containers. The VM is running behind an ha-proxy, which was used to route the traffic to the different services. The VM is running Ubuntu 22.04 LTS.
+
+## Repos in the deployment
+
+The repos used in the deployment are the following:
+1. https://github.com/GenomicDataInfrastructure/starter-kit-storage-and-interfaces.git
+2. https://github.com/GenomicDataInfrastructure/starter-kit-htsget 
+3. https://github.com/GenomicDataInfrastructure/starter-kit-rems
+4. https://github.com/GenomicDataInfrastructure/starter-kit-containerized-computation
+5. https://github.com/GenomicDataInfrastructure/starter-kit-beacon2-ri-api.git 
+
+# User facing services
+The user facing services are the following:
+1. Download
+2. Htsget
+3. REMS
+4. Login/Auth
+5. Inbox
+6. Beacon
+7. Public key endpoint
+
+The ports for these services should bee open, in order to be able to access them from outside the VM. The ports in our case are:
+1. Download: 8443
+2. Htsget: 8081
+3. REMS: 3000
+4. Login/Auth: 8080
+5. Inbox: 8000
+6. Beacon: 8085 TODO: Change to the actual port
+7. Public key endpoint: stored at an S3 bucket in our case
```

Comment:
This is not a port, right?

Reply:
The public key is not a port is it is categorized under the section `ports` as 7th point.

Reply:
OK